### PR TITLE
Update product-management-team-handbook.md

### PR DESCRIPTION
**Pull request by wiersgallak**
Added Core areas of Responsibilities for PMs

### CODE REVIEW

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 1]
```diff
@@ -13,14 +13,23 @@ The purpose of this handbook is to provide more details on the Product Manager r
...
+| Strategy Alignment | PDetermine how problem statements align with direction and mission of your area of ownership and strategy of Mattermost.  Determine priority of solving the problem and the scope at which it needs to be solved.  Drive alignment and agreement on importance of priority. Say no when something does not align. | Score and prioritize features by impact, Gain alignment from MLT, teammates, and GTM on the priority and scope of the required solution,Document and share decisions and alignment discussions|
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
| Strategy Alignment | Determine how problem statements align with direction and mission of your area of ownership and strategy of Mattermost. Determine priority of solving the problem and the scope at which it needs to be solved. Drive alignment and agreement on importance of priority. Say "no" when something does not align. | Score and prioritize features by impact, gain alignment from MLT, teammates, and GTM on the priority and scope of the required solution. Document and share decisions and alignment discussions. |
```

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 2]
```diff
@@ -13,14 +13,23 @@ The purpose of this handbook is to provide more details on the Product Manager r
...
 
+### Core Areas of Responsibility
+
+| Responsibility | What does this mean?| Actions taken for this responsibility|
+| :--- | :--- | :--- |
+| Problem Definition| Listen to current, prospective, and target customers and users to understand their problems and other solutions available to address their problems. Have a deep understanding of what the problem is, their use case, and what is a “must have” to resolve the problem and what is a “nice to have”. Understand where the problem is with the deep context of a user/customer experience. | Document problem statements and use cases from interviews and feedback channels, Conduct competitive analysis and market research, Determine demand for a solution by validating propensity of problem or use case within our broader customer/user bases and market, Document what is required by users or customers to solve the problem, Draft 1-pagers & PRFAQs|
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
| Problem Definition | Listen to current, prospective, and target customers and users to understand their problems and other solutions available to address their problems. Have a deep understanding of what the problem is, their use case, what's a “must have” to resolve the problem and what's a “nice to have”. Understand where the problem is with the deep context of a user/customer experience. | Document problem statements and use cases from interviews and feedback channels. Conduct competitive analysis and market research. Determine demand for a solution by validating propensity of problem or use case within our broader customer/user bases and market. Document what is required by users or customers to solve the problem. Draft 1-pagers and PRFAQs. |
```

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 3]
```diff
@@ -13,14 +13,23 @@ The purpose of this handbook is to provide more details on the Product Manager r
...
+| Strategy Alignment | PDetermine how problem statements align with direction and mission of your area of ownership and strategy of Mattermost.  Determine priority of solving the problem and the scope at which it needs to be solved.  Drive alignment and agreement on importance of priority. Say no when something does not align. | Score and prioritize features by impact, Gain alignment from MLT, teammates, and GTM on the priority and scope of the required solution,Document and share decisions and alignment discussions|
+| Solution Definition & Validation| Work with design and engineering to find solutions. Represent the voice of the user/customer to ensure must-have requirements are addressed. Coordinate external functions that need to be consulted in definition of solutions. | Write spec documents and produce low fidelity wireframes and user flows, Determine packaging based on how the solution fits the needs of ICs, Managers, or Executives, Validate solutions with users and customers to ensure they meet their use cases and requirements|
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
| Solution Definition & Validation | Work with design and engineering to find solutions. Represent the voice of the user/customer to ensure must-have requirements are addressed. Coordinate external functions that need to be consulted in definition of solutions. | Write spec documents and produce low fidelity wireframes and user flows. Determine packaging based on how the solution fits the needs of ICs, Managers, or Executives. Validate solutions with users and customers to ensure they meet their use cases and requirements. |
```

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 4]
```diff
@@ -13,14 +13,23 @@ The purpose of this handbook is to provide more details on the Product Manager r
...
+| Solution Scheduling & Development | Work with developers, designers and QA team to negotiate scope of functionality, resolve challenges, and ensure features meet quality standards. Represent resources and be the coordinator of collaboration across all functions and Eng teams. | Expose and resolve challenges between functions, Test features to ensure key use cases are solved, Update stakeholders with expected timelines and changes to solution scope|
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
| Solution Scheduling & Development | Work with developers, designers, and QA team to negotiate scope of functionality, resolve challenges, and ensure features meet quality standards. Represent resources and be the coordinator of collaboration across all functions and Eng teams. | Expose and resolve challenges between functions. Test features to ensure key use cases are solved. Update stakeholders with expected timelines and changes to solution scope. |
```

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 5]
```diff
@@ -13,14 +13,23 @@ The purpose of this handbook is to provide more details on the Product Manager r
...
+| Solution Release | Work with Release Management and Marketing to prepare customer and internal enablement and announcement materials.  Ensure go-to-market and operations are prepared to accommodate adoption of new functionality.  | Prepare release announcement from finalized 1-pagers/PRFAQs, record a demo of the feature, update public facing roadmaps and forums with release updates, Complete product documentation and changelog information with assistance from Tech Writers, Ready operational and customer facing teams with training and enablement materials |
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
| Solution Release | Work with Release Manager and Marketing to prepare customer and internal enablement and announcement materials. Ensure go-to-market and operations are prepared to accommodate adoption of new functionality. | Prepare release announcement from finalized 1-pagers/PRFAQs, record a demo of the feature, update public facing roadmaps and forums with release updates. Complete product documentation and changelog information with assistance from Tech Writers. Ready operational and customer-facing teams with training and enablement materials. |
```

File: operations/research-and-development/product/product-management-team-handbook.md [hunk 6]
```diff
@@ -13,14 +13,23 @@ The purpose of this handbook is to provide more details on the Product Manager r
...
+| Solution Adoption | Advocate for adoption of the new value added to the product. Monitor feedback and iterate on improvements to improve experience.  | Track feature usage and feedback through qualitative and quantitative data gathering, Demo and share features with existing customers and users, Iterate on feature with additional feedback from customers/users|
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-management-team-handbook.md]:
```suggestion
| Solution Adoption | Advocate for adoption of the new value added to the product. Monitor feedback and iterate on improvements to improve experience.  | Track feature usage and feedback through qualitative and quantitative data gathering. Demo and share features with existing customers and users. Iterate on feature with additional feedback from customers/users. |
```

**Review by justinegeffen — COMMENTED** [review]
This is awesome, thanks @wiersgallak! I've modified some of the content of the tables in terms of minor formatting. Wondering if it may be a good idea to bullet point within the table?

**Comment by wiersgallak** [comment]
@justinegeffen I was not sure how to do bullets in a table.  If you can help me with the syntax, I can update the content.  Thanks for your revisions.

**Review by jasonblais — DISMISSED** [review]

**Comment by wiersgallak** [comment]
Figured out the formatting.  Queuing for re-review.

**Review by jasonblais — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**